app-land/scss/_typography.scss

```diff
@@ -1,5 +1,5 @@
 body {
-  font-family: "Poppins", sans-serif;
+  font-family: $FontFamily;
```

Comment:
it's not based on code convention, how many times I have to repeat it ?! when do you want to learn it?